sdk/openai/azure-ai-openai-assistants/src/main/java/com/azure/ai/openai/assistants/models/VectorStoreChunkingStrategyResponse.java

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.assistants.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * An abstract representation of a vector store chunking strategy configuration.
+ */
+@Immutable
+public class VectorStoreChunkingStrategyResponse implements JsonSerializable<VectorStoreChunkingStrategyResponse> {
```

Comment:
Do we need this class? It's just a wrapper around the enum type.

Reply:
This is a polymorphic type. In the case where the `type` field is `static` there is an additional property in the JSON called `static` that needs to be accounted for.